### ISSUE TITLE
__main__.py that says what is installed

### ISSUE THREAD

**Issue by sushain97** [apertium/apertium-python#18]

**Comment by arghyatiger** [comment]
You mean when one does ``` import apertium ``` it should print all the modes installed?

**Comment by sushain97** [comment]
No, that's not what main.py is for.

**Comment by numan98khan** [comment]
I want to work on this issue. What I want to understand is that is there further description of the problem or am I missing something? What installation must be reported by __main__.py?

**Comment by sushain97** [comment]
Have you read https://stackoverflow.com/questions/4042905/what-is-main-py ?

**Comment by sushain97** [comment]
No, not really. That would be circular. It's just to show some basic
information about what's available.

**Comment by Vaydheesh** [comment]
Can you elaborate about what needs to be printed

**Comment by sushain97** [comment]
Something similar to what gets printed on apertium -l. Just the installed modes is good enough. Nothing fancy.
